woocommerce/payment-gateway/apple-pay/class-sv-wc-payment-gateway-apple-pay-frontend.php

```diff
@@ -177,12 +180,15 @@ protected function enqueue_js_handler( array $args, $object_name = '', $handler_
 	 * Concrete implementations can override this with their own handler.
 	 *
 	 * @since 5.6.0
+	 * @deprecated x.y.z
 	 *
 	 * @return string
 	 */
 	protected function get_js_handler_name() {
 
-		return 'SV_WC_Apple_Pay_Handler_5_6_1';
+		wc_deprecated_function( __METHOD__, 'x.y.z', parent::get_js_handler_class_name() );
```

Comment:
We are currently calling `get_js_handler_class_name()` and passing the result as the third argument of `wc_deprecated_function `. Should we pass `__CLASS__ . ':: get_js_handler_class_name()'` instead?

```suggestion
		wc_deprecated_function( __METHOD__, 'x.y.z', __CLASS__ . ':: get_js_handler_class_name()' );
```